charts/gitops-runtime/values.yaml

```diff
@@ -191,7 +191,7 @@ sealed-secrets:
   image:
     registry: 'quay.io'
     repository: 'codefresh/sealed-secrets-controller'
-    tag: '0.28.0'
+    tag: '0.29.0'
```

Comment:
please check why we set it explicitly -- it should be taken just from chart. Probably it was a workaround because the latest chart didn't contain the version